equipy/metrics/_performance_metrics.py

```diff
@@ -1,18 +1,22 @@
+"""Computation of the performance (i.e. measurement of the similarity between prediction and actual value)."""
+
+import numpy as np
+from typing import Callable
 from sklearn.metrics import mean_squared_error
 from ..utils.checkers import _check_metric
 
 
-def performance(y_true, y_pred, metric=mean_squared_error):
+def performance(y_true: np.ndarray, y_pred: np.ndarray, metric: Callable = mean_squared_error) -> float:
     """
     Compute the performance value for predicted fair output compared to the true labels.
 
     Parameters
     ----------
-    y_true : array-like
-        True labels or ground truth values.
-    y_pred : array-like
+    y_true : np.ndarray
+        Actual values.
+    y_pred : np.ndarray
         Predicted (fair or not) output values.
-    metric : callable, optional
+    metric : Callable, (default=mean_squared_error)
         The metric used to compute the performance, default=sklearn.metrics.mean_square_error.
```

Comment:
Add how it expects the input (ie that it expects y_true, y_pred)
depending on the metric used this might change